docs_outdoornav_user_manual/getting_started/tf_validation.mdx

```diff
@@ -0,0 +1,73 @@
+---
+title: TF Validation
+sidebar_label: TF Validation
+sidebar_position: 4
+toc_min_heading_level: 2
+toc_max_heading_level: 4
+---
+
+:::note
+
+Sensor TF validation should only be required if the performance is
+noticeably poor, such as when the UGV drifts off of the planned path or
+oscillations occur around the path).
+
+:::
+
+The coordinate transformation of the two GPS antennas, the IMU, the 2D
+and 3D Lidars to the base_link of the UGV should have already been set
+prior to receiving the UGV. However, ensure that they are set correctly.
+
+The ROS coordinate frame base_link, shown in the figure below, is
+located at the center of the bottom plate of the UGV. The environment
+variables below should be taken with respect to this coordinate frame.
+The X axis is in red (and pointing towards the front of the UGV), Y in
+green (pointing towards the left of the UGV) and Z in blue (pointing
+towards the sky).
+
+You can check the current value of the environment variables by running
+the following (and setting <variable_name\> to the names listed in the
+table below).
+
+``` bash
+printenv | grep <variable_name>
+```
+
+The environment variables for the position and orientation of each
+sensor are provided in the table below.
+
+_OutdoorNav sensor position and orientation environment variables_
+
+| Environment Variable | Function                              |
+|----------------------|---------------------------------------|
+| POSITION_GPS_XYZ | [X,Y,Z] position, in meters, of the position (rear) GPS antenna with respect to the base_link coordinate frame. |
+| POSITION_GPS_RPY | [Roll,Pitch,Yaw], in radians, of the position (rear) GPS antenna with respect to the base_link coordinate frame. |
+| HEADING_GPS_XYZ | [X,Y,Z] position, in meters, of the heading (front) GPS antenna with respect to the base_link coordinate frame. |
+| HEADING_GPS_RPY | [Roll,Pitch,Yaw], in radians, of the heading (front) GPS antenna with respect to the base_link coordinate frame. |

```

Comment:
front/rear could be confusing here, especially on Husky Observer where it is left/right